ons_alpha/urls.py

```diff
@@ -14,6 +14,22 @@
 from ons_alpha.search import views as search_views
 from ons_alpha.utils.cache import get_default_cache_control_decorator
 
+from wagtail.contrib.routable_page.models import RoutablePageMixin, route
+from ons_alpha.bulletins.views import latest_bulletin_view, previous_releases_view
+
+class BulletinSeriesPage(RoutablePageMixin, Page):
```

Comment:
This is a page model. Models should live in their app's `model.py`. 

In this case, `ons_alpha/bulletins/models.py`
There is no need to separate latest/previous in a view module as the logic here is fairly simple.

Adding a new model should require Django migrations (so, `./manage.py makemigrations` in the container or if you're running it directly, then in your active venv)